### ISSUE TITLE
team_owner_id should not be required for heartbeats

### ISSUE THREAD

**Issue by stephennancekivell** [opsgenie/terraform-provider-opsgenie#34]
Hi, when I try to create a heartbeat I get an error saying that the team_owner_id field is required. This shouldnt be the case.

### Terraform Version
```
terraform-12.6 -v
Terraform v0.12.6
+ provider.opsgenie v0.2.3
```
### Affected Resource(s)
Please list the resources as a list, for example:
- opsgenie
- opsgenie_heartbeat

### Terraform Configuration Files
```hcl
provider "opsgenie" {
  version = "~> 0.2"
  //  api_key is provided by the environment variable
}

resource "opsgenie_heartbeat" "heartbeat" {
  name          = "test"
  interval      = 15
  interval_unit = "minutes"
  enabled       = var.heartbeat_enabled
  //  owner_team_id = "bla bla should not need"
}

```

### Expected Behavior
It should create the heartbeat without needing the owner_team_id field.

### Actual Behavior
It crashes saying I need the owner_team_id field.
```
terraform-12.6 apply -target=module....opsgenie_heartbeat.heartbeat
module....opsgenie_heartbeat.heartbeat: Refreshing state... [id=test]

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # module....opsgenie_heartbeat.heartbeat will be updated in-place
  ~ resource "opsgenie_heartbeat" "heartbeat" {
        alert_tags    = []
        enabled       = false
        id            = "test"
        interval      = 15
        interval_unit = "minutes"
        name          = "test"
      - owner_team_id = "b092a64b-f11a-4997-9627-c6d9385a67d2" -> null
    }

Plan: 0 to add, 1 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

module....opsgenie_heartbeat.heartbeat: Modifying... [id=..-staging-2]


Error: Invalid request. Owner team cannot be empty. 

  on ../shared-modules/../main.tf line 75, in resource "opsgenie_heartbeat" "heartbeat":
  75: resource "opsgenie_heartbeat" "heartbeat" {

```
Note the *Error: Invalid request. Owner team cannot be empty. *

### Steps to Reproduce
1. `terraform apply`

### References
In the docs it says the field is optional, as it should be. https://www.terraform.io/docs/providers/opsgenie/r/heartbeat.html#owner_team_id

**Comment by ffahri** [comment]
Thank you for filling up this issue. This happens due to our go-sdk 
https://github.com/opsgenie/opsgenie-go-sdk-v2/blob/master/heartbeat/request.go#L112

I will make the correct changes and update provider as well.

**Comment by ffahri** [comment]
@stephennancekivell i changed go-sdk-v2 and updated provider.
https://github.com/terraform-providers/terraform-provider-opsgenie/pull/35
This is fixed in v0.2.4  (release will be ready in today)
However you may not be create global heartbeats if your account switched new heartbeat
https://docs.opsgenie.com/docs/heartbeat-v1-to-v2-help-guide

**Comment by stephennancekivell** [comment]
Wow, thanks a lot! :)
